### PR TITLE
new updated function_base.py

### PR DESCRIPTION
**Pull request by aniketrox**
change in average function where array input of fraction.Fraction dataype got changed to numpy arrray by np.asanyarray function

### CODE REVIEW

**Review by rossbar — COMMENTED** [review]
It looks like this PR removes a lot of recent documentation improvements, which indicates the branch was not created correctly.

You should make sure that `main` on your fork is up-to-date with the `main` branch on numpy/numpy, create a new branch from that point (`git checkout -b`) then commit the changes you'd like to propose to that branch and push it up to create a PR.

**Comment by aniketrox** [comment]
> It looks like this PR removes a lot of recent documentation improvements, which indicates the branch was not created correctly.
> 
> You should make sure that `main` on your fork is up-to-date with the `main` branch on numpy/numpy, create a new branch from that point (`git checkout -b`) then commit the changes you'd like to propose to that branch and push it up to create a PR.

Now what I have to do sir?

**Comment by rossbar** [comment]
I would recommend to start by reading through the [development workflow](file:///home/ross/repos/numpy/doc/build/html/dev/index.html#development-process-summary) from the contribution guide.

File: numpy/lib/function_base.py [hunk 1]
```diff
@@ -3942,7 +3935,7 @@ def percentile(a,
         8. 'median_unbiased'
         9. 'normal_unbiased'
 
-        The first three methods are discontinuous.  NumPy further defines the
+        The first three methods are discontiuous.  NumPy further defines the
```
Comment (InessaPawson) [inline numpy/lib/function_base.py:3938]: The original spelling was correct.
```suggestion
        The first three methods are discontinuous.  NumPy further defines the
```

**Comment by InessaPawson** [comment]
@aniketrox, thank you for submitting your suggestions. However, I'm not convinced that they would add value to the existing documentation.

**Comment by seberg** [comment]
@InessaPawson the doc changes are an accident, not sure how they actually come about.

@aniketrox thanks for your interest in contributing, I hope you can find something else to do!  I am going to close this PR for now.
The later change may help since it just simplifies the code, but the `avg_as_array` at the beginning is much clearer as to the intention.

**Pull request closed without merging**